native/membrane_videocompositor_wgpu/src/compositor/videos.rs

```diff
@@ -0,0 +1,67 @@
+use wgpu::util::DeviceExt;
+
+use super::textures::{YUVPlane, YUVTextures};
+use super::Vertex;
+
+#[rustfmt::skip]
+const INDICES: [u16; 6] = [
+    0, 1, 3, 
+    1, 2, 3
+];
+
+pub struct InputVideo {
```

Comment:
Later on, we'll need video placment on the scene for things such as video transformations (rotation, scale, crop etc.), so maybe we should have something like `video.position` or `video.placment` with a struct holding information about video position on the scene in scene clip coordinates ([-1, 1] range), scene pixel coordinates (([0, scene.width], [0, scene.height]) range) and vertices?

Reply:
I think this makes more sense to be implemented in #37, since it makes the input videos use the pixel coordinates.

Reply:
Ok